specification/cognitiveservices/data-plane/Language/preview/2021-05-01-preview/questionanswering-authoring.json

```diff
@@ -472,7 +472,7 @@
         }
       }
     },
-    "/language/query-knowledgebases/projects/{projectName}/synonyms": {
```

Comment:
Just to double check, I assume this was done originally in error since the base path is already `{Endpoint}/language`, correct?

Reply:
Yes, earlier it was an error. Fixing it this time.